### PR TITLE
Animations: whitelist offset distance and regroup condition types

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: extensions/amp-animation/0.1/web-animation-types.js [hunk 1]
```diff
@@ -168,6 +168,8 @@ const WHITELISTED_RPOPS = {
   'opacity': true,
   'transform': true,
   'visibility': true,
+  'offset-distance': true,
```
Comment (alanorozco) [inline extensions/amp-animation/0.1/web-animation-types.js]: Why set this twice?

Comment (dvoytenko) [inline extensions/amp-animation/0.1/web-animation-types.js]: Both forms are currently allowed and we leave it up to the animations API to convert. We could convert ourselves, in which case we'd only need one.

Comment (dvoytenko) [inline extensions/amp-animation/0.1/web-animation-types.js]: I'll go ahead and merge. It's possible we'll convert properties in upcoming tasks.

Comment (alanorozco) [inline extensions/amp-animation/0.1/web-animation-types.js]: Thanks. I was just curious, PR is fine as is so I approved it.

**Pull request merged**